### PR TITLE
toml page - replace li by table

### PR DESCRIPTION
**Pull request by jmaitrehenry**
Related to #1993

### CODE REVIEW

File: docs/toml.md [hunk 1]
```diff
@@ -1708,10 +1712,16 @@ Træfik can be configured to use Amazon ECS as a backend configuration:
...
+# Cluster = "default"
+
+# ECS Clusters Name
+#
 # Optional
-# Default: "default"
+# Default: ["default"]
 #
-Cluster = "default"
+Clusters = ["default"]
```
Comment (marco-jantke) [inline docs/toml.md]: I am confused by this change. I can't see any notion of `Clusters` in the code. Am I missing something?

File: docs/toml.md [hunk 2]
```diff
@@ -1720,6 +1730,13 @@ Cluster = "default"
 #
 Watch = true
 
+# Enable auto discover ECS clusters
+#
+# Optional
+# Default: false
+#
+AutoDiscoverClusters = false
```
Comment (marco-jantke) [inline docs/toml.md]: The same as in my previous comment here.

File: docs/toml.md [hunk 3]
```diff
@@ -1780,6 +1797,8 @@ Træfik needs the following policy to read ECS information:
             "Sid": "Traefik ECS read access",
             "Effect": "Allow",
             "Action": [
+                "ecs:ListClusters",
```
Comment (marco-jantke) [inline docs/toml.md]: I don't see any calls to `ListClusters` or `DescribeClusters` in the ecs provider code.

File: docs/toml.md (hunk 1 above)
Comment (nmengin) [inline docs/toml.md]: In fact, the current branch seems to be out-of-date with the master.
Only he file `docs/toml.md` is up-to-date that's why these differences appear.

Comment (marco-jantke) [inline docs/toml.md]: Alright, then I will wait for the rebase and have a look again.

**Comment by jmaitrehenry** [comment]
@nmengin @marco-jantke I just push the rebase

In an another PR I will move more li to table if you want.

**Comment by ldez** [comment]
@jmaitrehenry Could you use a table formatting tool (like [tablesgenerator](http://www.tablesgenerator.com/markdown_tables) ?

**Comment by jmaitrehenry** [comment]
@ldez done

**Pull request merged**